cpp/src/io/utilities/column_buffer.hpp

```diff
@@ -108,6 +108,19 @@ struct column_buffer {
 
   auto& null_count() { return _null_count; }
 
+  /**
+   * @brief Sets the stream to be used for deallocation of internal buffers
+   */
+  void set_stream(rmm::cuda_stream_view stream) noexcept
+  {
+    if (_strings) _strings->set_stream(stream);
+    _data.set_stream(stream);
+    _null_mask.set_stream(stream);
+    for (auto& child : children) {
+      child.set_stream(stream);
+    }
+  }
```

Comment:
Same here. Move the function definition into `cpp/src/io/utilities/column_buffer.cpp` if possible.

Reply:
Done

Reply:
`set_stream()` still seems to be defined in the header `column_buffer.hpp`?

Reply:
I'm not sure how that happened. VS Code shows the file without the definition, but the local filesystem still had it. Updated via my old friend, vi.